app/controllers/dashboard.js

```diff
@@ -26,6 +27,10 @@ export default Ember.Controller.extend({
         return this.get('myFollowing').slice(0, TO_SHOW);
     }),
 
+    visibleStats: computed('myStats', function() {
```

Comment:
I was looking at the code, and this and the other `computed` that depend on arrays should be `myArray.[]`.
I wouldn't block the PR on that, but just want to note so it gets addressed.